README.md

```diff
@@ -41,21 +43,53 @@ generic with minimal need for project-specific behavior.
 The remaining infrastructure is managed internally with TerraForm.
 
 ### 1.3. Project configuration
+Getting started with out attempting to make major contributions, a set of the test suite passes without AWS credentials.
```

Comment:
Getting started without attempting to make contributions does not require AWS credentials. A subset of the test suite passes without configured AWS credentials. To validate your setup, we'll be running one of those tests at the end.

Reply:
Wrap paragraphs at exactly 80 columns.